python_modules/dagster/dagster/grpc/types.py

```diff
@@ -124,7 +124,7 @@ class ExecuteStepArgs(
     namedtuple(
         "_ExecuteStepArgs",
         "pipeline_origin pipeline_run_id step_keys_to_execute instance_ref "
-        "retry_mode known_state should_verify_step",
+        "retry_mode known_state should_verify_step should_start_background_cancellation_thread",
```

Comment:
`should_start_background_cancellation_thread` is very current-impl specific. I think from the API perspective the difference is between the run worker being responsible or this step worker being responsible for itself 
riffing:
`self_managed_cancellation: bool`
`cancellation_source: external (default) | self`